website/docs/providers/terraform/d/remote_state.html.md

```diff
@@ -22,20 +22,23 @@ use interpolations when configuring them.
 ## Example Usage
 
 ```hcl
+
 data "terraform_remote_state" "vpc" {
   backend = "atlas"
-  config {
+  config = {
     name = "hashicorp/vpc-prod"
   }
 }
 
 # Terraform >= 0.12
+
```

Comment:
can you please remove the added newlines, here and on line 41?

Reply:
```suggestion
```